scripts/setupenv.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+APPDIR=/srv/www/todo_app
```

Comment:
Given how little shared config there is here, personally I'd be inclined to split this out into three scripts, which you can explicitly name from your entrypoints